<launch>
  <arg name="urdf_filename" default="$(find galileo_go1_description)/urdf/go1.urdf" />
  <arg name="parameters_location" default="$(find galileo_go1_description)/SolverParameters/go1_solver_parameters.txt" />
  <arg name="solver_id" default="galileo_go1" />

  <param name="galileo_ros/urdf_filename" value="$(arg urdf_filename)"/>
  <param name="galileo_ros/parameters_location" value="$(arg parameters_location)"/>
  <param name="galileo_ros/solver_id" value="$(arg solver_id)"/>

  <node name="galileo_ros_legged_solver_node" type="galileo_ros_legged_solver_node" pkg="galileo_ros" output="screen">
  </node>
  
  <node name="galileo_ros_legged_initialization_node" type="galileo_ros_legged_initialization_node" pkg="galileo_ros" output="screen">
  </node>
  
</launch>